07-sampling.Rmd

```diff
@@ -213,15 +218,26 @@ Let's start by performing the virtual analog of the tactile sampling exercise we
 bowl
 ```
 
-```{r message=FALSE, echo=FALSE, purl=FALSE}
-# Dynamic coding of summary statistics for bowl i.e. avoid hard-coding any values
-# wherever possible
-num_balls <- nrow(bowl)
-num_red <- bowl %>%
-  summarize(red = sum(color == "red")) %>%
-  pull(red)
-prop_red <- num_red / num_balls
-percent_red_chr <- prop_red %>% percent(accuracy = 0.1)
+Observe that `bowl` has `r num_balls` rows, telling us that the bowl contains `r num_balls` equally sized balls. The first variable `ball_ID` is used as an *identification variable* as discussed in Subsection \@ref(identification-vs-measurement-variables); none of the balls in the actual bowl are marked with numbers. The second variable `color` indicates whether a particular virtual ball is red or white. View the contents of the bowl in RStudio's data viewer and scroll through the contents to convince yourself that `bowl` is indeed a virtual analog of the actual bowl in Figure \@ref(fig:sampling-exercise-1).
+
+Now that we have a virtual analog of our bowl, we now need a virtual analog to the shovel seen in Figure \@ref(fig:sampling-exercise-2) to generate virtual samples of 50 balls. We're going to use the `rep_sample_n()` function included in the `moderndive` package. This function allows us to take `rep`eated, or `rep`licated, `samples` of size `n`. 
```

Comment:
```suggestion
Now that we have a virtual analog of our bowl, we next need a virtual analog to the shovel seen in Figure \@ref(fig:sampling-exercise-2) to generate virtual samples of 50 balls. We're going to use the `rep_sample_n()` function included in the `moderndive` package. This function allows us to take `rep`eated, or `rep`licated, `samples` of size `n`. 
```

Reply:
Note that Simon has implemented a `rep_slice_sample()` function in {infer} that should be merged in soon: https://github.com/tidymodels/infer/pull/362/files. Might be best to add a reminder to convert all the `rep_sample_n()` over when the next version of {infer} goes to CRAN.